lib/requester.js

```diff
@@ -55,6 +56,33 @@ function getAuthenticatorKerberos(client) {
   return client.authenticatorKerberos;
 }
 
+function getAccessToken(operation, callback){
+
+  let postData = `${encodeURI('grant_type')}=${encodeURI('apikey')}&${encodeURI('key')}=${encodeURI(operation.client.connectionParams.apiKey)}`;
```

Comment:
Small thing - I believe `const` is generally preferred whenever possible to make it clear that something won't change, whereas `let` implies that the variable assignment may be altered later on. I recommend using `const` whenever possible - I think this is one of those things JS does better than Java, as Java requires an additional keyword of `final` whereas in JS, you just use `const` instead of `let` or `var`.

Reply:
Done.